sphinx-docs/index.rst.j2

```diff
@@ -47,6 +63,15 @@ The following section contains information intended to help developers
 understand the inner workings of the CALDERA adversary emulation tool, CALDERA
 plugins, or new tools that interface with the CALDERA server.
 
+.. toctree::
+   :maxdepth: 2
+   :caption: Development
```

Comment:
`Development` is pretty similar to `Developer Information`. Maybe rename this section `Developer Information` and the other one below something like `Python API` or `Caldera Core Python API`.

Reply:
updated to `Developer Information` and `Core System API`, I think that's more consistent with references to "the core system" at the top of the documentation page